packages/mui-styled-engine-sc/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/styled-engine-sc",
-  "version": "6.0.0-alpha.5",
+  "version": "6.0.0-alpha.6",
```

Comment:
Why are we updating this? There is no change even in it's dependencies.

Reply:
It was picked up by lerna, I haven't checked individually what needs to be updated.